backend/app/models/investor.py

```
@@ -0,0 +1,71 @@
+class InvestorProfileUpdate(SQLModel):
+    firm_name: Optional[str] = None
+    bio: Optional[str] = None
+    website: Optional[str] = None
+    location: Optional[str] = None
+    linkedin_url: Optional[str] = None
+    twitter_url: Optional[str] = None
+    investment_focus: Optional[List[Industry]] = None
+    preferred_stages: Optional[List[FundingStage]] = None
+
+
+class InvestorWithUserRead(SQLModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)